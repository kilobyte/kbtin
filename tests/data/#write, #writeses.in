#var foo a
#sub foo a
#ali foo a
#pro foo a
#ant foo
#ro foo bar e
#run a //selfpipe
#var bar a
#sub bar a
#ali bar a
#pro bar a
#ant bar
#ro bar baz e
#write f1
#writes f2
#sys grep -v message f1
#show -----
#sys grep -v message f2
#sys rm f1 f2
#end
