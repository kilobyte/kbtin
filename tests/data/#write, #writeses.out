#COMMANDS-FILE WRITTEN.
#DELTA COMMANDS FILE WRITTEN.
#alias {bar} {a}
#alias {foo} {a}
#promptaction {bar} {a} {5}
#promptaction {foo} {a} {5}
#antisub {bar}
#antisub {foo}
#sub {bar} {a}
#sub {foo} {a}
#var {bar} {a}
#var {foo} {a}
#route {foo} {bar} {e} 10
#route {bar} {baz} {e} 10
-----
#alias {bar} {a}
#promptaction {bar} {a} {5}
#antisub {bar}
#sub {bar} {a}
#var {bar} {a}
#route {bar} {baz} {e} 10

#SESSION 'a' DIED.
